app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLocked/index.tsx

```diff
@@ -40,17 +40,18 @@ const getSidebarItems = (
   return sidebarItems;
 };
 
+// TODO: fix this
 const probesetToMapping = (
   probesets:
-    | (getExperiment_experimentBySlug_primaryProbeSets | null)[]
-    | (getExperiment_experimentBySlug_secondaryProbeSets | null)[],
+    | (getExperiment_experimentBySlug_primaryOutcomes | null)[]
```

Comment:
This is from find-and-replace leftover pasta, `getExperiment_experimentBySlug_primaryOutcomes` isn't generated